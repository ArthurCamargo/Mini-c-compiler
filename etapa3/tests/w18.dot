digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 [label="a"];
n3;
n3 -> n4;
n3 -> n5;
n3 -> n6;
n4;
n4 [label="a"];
n5;
n5 [label="b"];
n6;
n6 -> n7;
n6 -> n8;
n7;
n7 [label="b"];
n8;
n8 [label="a"];
n6 [label="="];
n3 [label="="];
n1 [label="if"];
n0 [label="f"];

}