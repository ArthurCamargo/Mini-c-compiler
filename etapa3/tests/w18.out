0x55eb051e9350
0x55eb051e9350, 0x55eb051e9770
0x55eb051e9770
0x55eb051e9770, 0x55eb051e94c0
0x55eb051e9770, 0x55eb051e95f0
0x55eb051e94c0
0x55eb051e94c0 [label="a"];
0x55eb051e95f0
0x55eb051e95f0, 0x55eb051e9530
0x55eb051e95f0, 0x55eb051e95a0
0x55eb051e95f0, 0x55eb051e9720
0x55eb051e9530
0x55eb051e9530 [label="a"];
0x55eb051e95a0
0x55eb051e95a0 [label="b"];
0x55eb051e9720
0x55eb051e9720, 0x55eb051e9660
0x55eb051e9720, 0x55eb051e96d0
0x55eb051e9660
0x55eb051e9660 [label="b"];
0x55eb051e96d0
0x55eb051e96d0 [label="a"];
0x55eb051e9720 [label="="];
0x55eb051e95f0 [label="="];
0x55eb051e9770 [label="if"];
0x55eb051e9350 [label="f"];
